src/main/scala/Day6/HashMapAnagrams.scala

```diff
@@ -0,0 +1,19 @@
+package Day6
+
+import scala.io.Source
+
+class HashMapAnagrams {
+
+def storeLinesInArray(filename: String) = {
+  val source = Source.fromFile(filename)
+  val linesArray = source.getLines.toArray
+  linesArray
```

Comment:
the whole method body could be this one line statement `Source.fromFile(filename).getLines.toArray` although this is not a safe way to read from file as you're not handling any errors, what if the path you give is broken, or the file isn't there

Reply:
I am going to start this exercise again